internal/pwru/output.go

```diff
@@ -274,7 +274,20 @@ func getExecName(pid int) string {
 	return execName
 }
 
-func getTupleData(event *Event, outputTCPFlags bool) (tupleData string) {
+func getAddrByArch(event *Event, o *output) (addr uint64) {
+	switch runtime.GOARCH {
+	case "amd64":
+		addr = event.Addr
+		if event.Type == eventTypeKprobe {
+			addr -= 1
+		}
+	case "arm64":
+		addr = event.Addr
+	}
+	return addr
+}
```

Comment:
Nack. 

With current enhancements, this function is useless and can be removed.

Reply:
Sorry, could you?
```
git fetch -u origin
git rebase origin/main # remove the function
git push -f
```
I don't have permissions to push to your fork.